third_party/txt/src/txt/paragraph.cc

```diff
@@ -43,10 +43,31 @@
 namespace txt {
 namespace {
 
-const sk_sp<SkTypeface>& GetTypefaceForGlyph(const minikin::Layout& layout,
-                                             size_t index) {
+class GlyphTypeface {
+ public:
+  GlyphTypeface(const sk_sp<SkTypeface>& typeface, minikin::FontFakery fakery)
```

Comment:
`const sk_sp<SkTypeface>&` and then copying is odd. `sk_sp<SkTypeface>` with an `std::move` into the ivar.

Reply:
done